src/mixins/payment-mixin.ts

```diff
@@ -159,6 +160,9 @@ export class PaymentMixin extends Mixins(ActionMixin) {
       const parsedAccountInfo = JSON.parse(accountInfo)
       headers['Account-Id'] = parsedAccountInfo.id
     }
+    if (this.getStaffPayment.option === StaffPaymentOptions.NONE) {
```

Comment:
I don't understand this. Why only set FN when staff payment is none? Is this because it doesn't apply when staff are doing the payment? If so then maybe there's a more clear (and without assuming the option) way to do this. Or add a comment saying what you're doing (so it looks like you are doing this explicitly/on purpose).

Reply:
This is because in buildStaffPayment we set the folioNumber to the headers. If it's not StaffPayment we need to set it to headers. Maybe we'd better refactor this buildStaffPayment function's name to buildHeaders and include all the header related logic to it. What do you think?

Reply:
Yes, that would make more sense :)